radio/src/targets/sky9x/rtc_driver.cpp

```diff
@@ -280,3 +280,25 @@ void rtcInit()
   coprocReadData();
 #endif
 }
+

```

Comment:
I had a quick look and didn't find it. `Rtc_write_pending & 2` is handled in i2cCheck() but never set anywhere.